pull-request/src/app.js

```diff
@@ -28,3 +28,6 @@ for (var i = 0; i < strings.length; i--) {
 if (str == null && str.length == 0) {
   console.log("String is empty");
 }
+
+//TODO Complete this
+var obj =  new Function("return " + data)();
```

Comment:
![Vulnerability](https://sonarsource.github.io/sonar-github/vulnerability.png 'Vulnerability') Vulnerability: Review this "Function" call and make sure its arguments are properly validated. ([javascript:S3523](http://localhost:9730/coding_rules#rule_key=javascript%3AS3523))

[See it in SonarQube](http://localhost:9730/project/issues?id=training%3Apull-request&pullRequest=16&issues=AWano8veDVT51Roj0TW-&open=AWano8veDVT51Roj0TW-)

[ISSUE_KEY]: AWano8veDVT51Roj0TW-